### PR TITLE
Feat/add dynamoose functions

### PR DESCRIPTION
**Pull request by marcosalves-bitg**
This PR adds the functions to do CRUD with dynamoose

### CODE REVIEW

**Review by Viserion77 — DISMISSED** [review]
LGTM BRABO DE MAIS

File: src/dynamoose/dynamoose.ts [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+const postDefaultFunction = async (
+  crudInputParams: DynamooseCrudInputParams,
+  dynamoObjectKeys: DynamoObjectKeys,
+  event?: DynamooseEventFunctionType
+): Promise<DynamooseOutputParams> => {
+  const updatedCrudInputParams: DynamooseCrudInputParams = event ? await event(crudInputParams) : crudInputParams
+
+  return {
+    method: 'create',
+    dynamooseData: assembleEntityData(updatedCrudInputParams, dynamoObjectKeys)
```
Comment (Viserion77) [inline src/dynamoose/dynamoose.ts:67]:
```suggestion
    dynamooseData: assembleEntityData(updatedCrudInputParams, dynamoObjectKeys, updatedCrudInputParams.entity)
```

File: src/dynamoose/dynamoose.ts [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+const putDefaultFunction = async (
+  crudInputParams: DynamooseCrudInputParams,
+  dynamoObjectKeys: DynamoObjectKeys,
+  event?: DynamooseEventFunctionType
+): Promise<DynamooseOutputParams> => {
+  const updatedCrudInputParams: DynamooseCrudInputParams = event ? await event(crudInputParams) : crudInputParams
+
+  return {
+    method: 'update',
+    dynamooseData: assembleEntityData(updatedCrudInputParams, dynamoObjectKeys)
```
Comment (Viserion77) [inline src/dynamoose/dynamoose.ts:80]:
```suggestion
    dynamooseData: assembleEntityData(updatedCrudInputParams, dynamoObjectKeys, updatedCrudInputParams.entity)
```

File: src/dynamoose/dynamoose.ts [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+const deleteDefaultFunction = async (
+  crudInputParams: DynamooseCrudInputParams,
+  dynamoObjectKeys: DynamoObjectKeys,
+  event?: DynamooseEventFunctionType
+): Promise<DynamooseOutputParams> => {
+  const updatedCrudInputParams: DynamooseCrudInputParams = event ? await event(crudInputParams) : crudInputParams
+
+  return {
+    method: 'delete',
+    dynamooseData: assembleEntityKeys(updatedCrudInputParams, dynamoObjectKeys)
```
Comment (Viserion77) [inline src/dynamoose/dynamoose.ts:93]:
```suggestion
    dynamooseData: assembleEntityData(updatedCrudInputParams, dynamoObjectKeys)
```

**Review by Viserion77 — CHANGES_REQUESTED** [review]

**Review by Viserion77 — APPROVED** [review]
LGTM SEMPRE ACREDITEI

**Review by ricardobaehr — APPROVED** [review]
LGTM

**Pull request merged**